### DISCUSSION TITLE
Bug, I created two habit projects. When adding widgets on the mobile desktop, the two projects overlap and cannot be separated. How can I operate

### DISCUSSION THREAD

**Discussion by BraveMomo** [iSoron/uhabits#1233] [Help & FAQ]
## Pre-submission checklist
- [ ] I am submitting a bug report, not a feature request.
- [ ] I have searched for similar issues, but did not find any matches.

## Description
A clear and concise description of what the problem is.

## Steps to reproduce
1. Go to ...
2. Click on ....
3. Scroll down to ....

## System information
 - Phone: [e.g. Google Pixel 4]
 - Phone Operating System: [e.g. Android 10]
 - App version: [e.g. 2.0.2]
![%@CJ MFU`}CLV}SQO0XV~OR](https://user-images.githubusercontent.com/22335503/144980829-56193e9c-3ca2-4153-8c21-8e387d62042a.png)

## Screenshots

Bug, I created two habit projects. When adding widgets on the mobile desktop, the two projects overlap and cannot be separated. How can I operate

**Comment by hiqua** [comment]
Delete the widget and recreate it. You can also swipe it to have access to the other habit.

**Reply by niktss** [reply]
I think the stacked checkmark widget is a great idea, but a quick tutorial that we are supposed to swipe up and down on it would be appreciated. I initially thought it was a glitched widget and tried to resize it to fix it, but when the layout didn't change I set out to 'figure out the design'. This doesn't need to happen and will improve usability
